anchore_engine/analyzers/syft/__init__.py

```diff
@@ -19,6 +19,16 @@ def catalog_image(image):
     nested_dict = lambda: collections.defaultdict(nested_dict)
     findings = nested_dict()
 
+    # This is the only use case for consuming the top-level results from syft,
```

Comment:
Should this be placed in the handlers package just continuity?

Reply:
That is a great question. If I did that, I would still need to import it and call it here because it is needed just once. Is that what you would prefer? There is no other handling at all for busybox

Reply:
Nope, that works for me.

Reply:
Your point is still valuable though, and I think we should create a `busybox` module if this is no longer a "one of".